confmodel/tests/test_fields.py

```diff
@@ -145,7 +145,7 @@ def assert_url(value,
         assert_url(self.field_value(field, 'foo'), path='foo')
         assert_url(self.field_value(field, u'foo'), path='foo')
         assert_url(self.field_value(field, u'foo\u1234'),
-                   path='foo\xe1\x88\xb4')
+                   path=u'foo\u1234')
```

Comment:
We use this code in lots of places with Twisted. Twisted's URL handling code (like treq) gets really unhappy when given unicode strings for URLs. We'll need to see how deep this rabbit hole goes because these things are difficult to test across all the places it's being used.